content/references/rippled-api/api-conventions/currency-formats.md

```diff
@@ -48,11 +55,12 @@ The standard currency mapping allocates the bits as follows:
 ![Standard Currency Code Format](img/currency-code-format.png)
 
 1. The first 8 bits must be `0x00`.
-2. The next 96 bits are reserved, and should be all `0`s.
+2. The next 88 bits are reserved, and should be all `0`'s.
 3. The next 24 bits represent 3 characters of ASCII.
     Ripple recommends using [ISO 4217](http://www.xe.com/iso4217.php) codes, or popular pseudo-ISO 4217 codes such as "BTC". However, any combination of the following characters is permitted: all uppercase and lowercase letters, digits, as well as the symbols `?`, `!`, `@`, `#`, `$`, `%`, `^`, `&`, `*`, `<`, `>`, `(`, `)`, `{`, `}`, `[`, `]`, and <code>&#124;</code>. The currency code `XRP` (all-uppercase) is reserved for XRP and cannot be used by issued currencies.
-4. The next 8 bits indicate the currency version. If the same currency is reissued with a different value, you can increment this value to keep the currencies separate.
-5. The next 24 bits are reserved and should be all `0`s.
+4. The next 40 bits are reserved and should be all `0`'s.
+
+In the rare case that a field specifies a currency code for XRP, the currency code should be all zeroes.
```

Comment:
I'd probably strike the work "rare". We either support the format or we don't. A currency code of all zeros is XRP.

Reply:
It is rare that a field specifies a currency code for XRP, though. The only case I can think of is path steps. I was hoping to head off confusion about any time you specify an XRP amount that you don't actually provide a currency code there (because the "not XRP" bit of the value handles it).

Reply:
Yeah, I see where you're comming from. OK, I'm fine with keeping "rare".